sourcegraph/sourcegraph
# Code Insights: Add drilldown search context filter

Closes: https://github.com/sourcegraph/sourcegraph/issues/33249

## Test plan
- Make sure that the context filter in drill down panel works properly
- Make sure that you can save filters
- Make sure that you can create insight with drill-down filter creation UI
- Make sure that drill-down panel's layout looks like it's presented in [designs](https://www.figma.com/file/Ml5dtV9ni4mfffbfOfF9jg/Allow-search-contexts-as-filters-to-code-insights-(v0)-%2333249?node-id=246%3A11) 


## App preview:

- [Link](https://sg-web-vk-add-drilldown-context-filter.onrender.com)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
